bumble/device.py

```diff
@@ -2110,11 +2110,29 @@ async def update_connection_parameters(
         supervision_timeout,
         min_ce_length=0,
         max_ce_length=0,
-    ):
+        use_l2cap=False,
+    ) -> bool:
```

Comment:
I don't think there's a need to return a boolean here. This function either just returns (None), or raises an exception.